src/Groups/abelian_aut.jl

```diff
@@ -0,0 +1,89 @@
+function _isomorphic_gap_group(A::GrpAbFinGen; T=PcGroup)
+  # find independent generators
+  if isdiagonal(rels(A))
+    exponents = [Int(a) for a in diagonal(rels(A))]
```

Comment:
Are all the exponents necessarily "small", i.e., fit into an `Int`? Perhaps just use `diagonal(rels(A))` directly?
```suggestion
    exponents = diagonal(rels(A))
```

Reply:
The reason is that `abelian_group` does not accept fmpz:
```

julia> abelian_group(PcGroup, [1, 23])
<pc group of size 23 with 2 generators>

julia> abelian_group(PcGroup, fmpz[1, 23])
ERROR: MethodError: no method matching abelian_group(::Type{PcGroup}, ::Vector{fmpz})

```

Reply:
Ouch. That's an oversight / bug. I'll log this as an issue and try to resolve it later today (after teaching)